sklearn/dummy.py

```diff
@@ -352,6 +353,20 @@ def score(self, X, y, sample_weight=None):
             X = np.zeros(shape=(len(y), 1))
         return super().score(X, y, sample_weight)
 
+    @property
+    def n_features_in_(self):
```

Comment:
same here, we could set the attribute to `None` in fit, and then it all works out. But this also looks good to me.